### ISSUE TITLE
Request - Add location names to your website.

### ISSUE THREAD

**Issue by HaydarOzturk** [sebastienvercammen/Pokemon-Go-Coords#5]
When you pull the coordinates, can't you arrange something like auto-naming into the website? At least the city and the country that they are in. Mybe we could be able to choose cities.

**Comment by sebastienvercammen** [comment]
@HaydarOzturk Do you feel location filtering would be a good addition to the project?

Most of the project's users are running automated bots that snipe all of the locations rather than only those around them. Adding a location filter might not be any help to them.

**Comment by HaydarOzturk** [comment]
Yes but like additional thing for only choice option mybe. This way we could know about nests and regional locations. Mybe a saperated little code will do the trick. They eventually are gonna start banning people because they jump too much so I don't think many people does that big jumps as you pridected :)
